src/splat/scripts/split.py

```diff
@@ -585,6 +535,12 @@ def add_arguments_to_parser(parser: argparse.ArgumentParser):
         help="Disasemble matched functions and migrated data",
         action="store_true",
     )
+    parser.add_argument(
+        "-I",
+        help="Add the directory to the list of search directories when including other config",
```

Comment:
Could you add a long option too?
```suggestion
    parser.add_argument(
        "--include",
        "-I",
        help="Add the directory to the list of search directories when including other config",
```

Reply:
Added.

Reply:
This longopt name is based on the clang long name for the similar concept.